text/0016-katello-as-a-plugin/0017-merge-katello-docs.md

```diff
@@ -0,0 +1,46 @@
+- Feature Name: Merge Katello documentation to theforeman.org
+- Start Date: 2016-10-11
+- RFC PR: (leave this empty)
+- Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Today the Foreman website and plugins are kept in a different repository than Katello. Further, the websites and documentation are at different URLs http://www.theforeman.org and http://www.katello.org.
+
+# Motivation
+[motivation]: #motivation
+
+The separation of documentation serves to push users into feeling there is a difference between Foreman and Katello. And while there is some technical differences currently, the goal is to bring everything in to alignment. One way to do this is by having a central place to go for documentation on all things Foreman and plugins. The expectation is that users will go to theforeman.org and be able to find Katello plugin documentation as you would any other plugins.
+
+# Detailed design
+[design]: #detailed-design
+
+Here are the set of action items for achieving this:
+
+ * Set a redirect from katello.org to theforeman.org Katello plugin page
+ * Carry forward Katello 2.4+ documentation versions
+ * Add a Troubleshooting guide to theforeman.org and include info from katello.org
```

Comment:
+1

I think a top-level troubleshooting link from https://theforeman.org/documentation.html would be helpful.  It's kind of spread out now.

A few plugins have their own Troubleshooting guides, foreman has this page: https://theforeman.org/manuals/1.12/index.html#7.Troubleshooting

But most things tend to go here:
  http://projects.theforeman.org/projects/foreman/wiki/Troubleshooting

and here:
  http://projects.theforeman.org/projects/foreman/wiki/ErrorCodes